src/widgets/table/table.rs

```diff
@@ -1871,4 +2147,52 @@ mod tests {
                 .remove_modifier(Modifier::CROSSED_OUT)
         );
     }
+
+    use rstest::rstest;
```

Comment:
It would be nice to move this (and other `rstest` imports) to the top of the tests module.

Reply:
Moved this one. Not sure how to move the other rstest imports since they are in separate modules.

Reply:
Yup, I think it would be better to import them from the top-level tests module instead of explicitly every time. Not a biggie though.

Reply:
Done :+1: